File: unit-3/sprint-3/day-3/you/intentJObservice/app/src/main/java/com/example/jobservice/MainActivity.java
package com.example.jobservice;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Context;
import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity implements View.OnClickListener {
    private Button mbtnclick1;
    private Button mbtnclick2;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        mbtnclick1=findViewById(R.id.btn1);
        mbtnclick2=findViewById(R.id.btn2);

    }


    @Override
    public void onClick(View v) {
        int id=v.getId();
        switch (id){
            case R.id.btn1:
                Intent intent1=new Intent(MainActivity.this,JobService.class);
                intent1.putExtra("jobId","1");
                JobService.enqueueWork(MainActivity.this,intent1);
                break;
            case R.id.btn2:
                Intent intent2=new Intent(MainActivity.this,JobService.class);
                intent2.putExtra("jobId","2");
                JobService.enqueueWork(MainActivity.this,intent2);
                break;

        }
    }
}

File: unit-3/sprint-4/day-3/we/kotlin2/app/src/main/java/com/example/kotlinwe2/MainActivity.kt
package com.example.kotlinwe2

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        launchfrag()
    }

    private fun launchfrag(){
        val fragmentmanager=supportFragmentManager
        val fragmenttransation=fragmentmanager.beginTransaction()
         fragmenttransation.add(R.id.container,UserDetailFragment(),"userDetailFragment").commit()

    }
}